library/firewall_lib.py

```diff
@@ -161,6 +161,17 @@
 
 from ansible.module_utils.basic import AnsibleModule
 from distutils.version import LooseVersion
+from subprocess import check_output
+import hashlib
+import shutil
+import os
+import glob
+
+CONFIGS_TO_DELETE = [
+    "/etc/firewalld/zones/*",
+    "/etc/firewalld/policies/*",
+    "/etc/firewalld/direct.xml",
+]
```

Comment:
What about `/etc/firewalld/firewalld.conf`? That's where the default zone setting goes.